### ISSUE TITLE
Pre-training Image-Text Matching

### ISSUE THREAD

**Issue by ZhiweiXie1** [salesforce/ALBEF#82]
Hello! Thank you in advance, I have a question to ask.
![image](https://user-images.githubusercontent.com/33353151/167263225-4ee973e8-94f5-4cda-b3e3-2447bf987635.png)
There are two classification cases in the paper, matching and non-matching. 
![image](https://user-images.githubusercontent.com/33353151/167263301-2c8f8dbd-d44c-47c0-bbeb-8b2b26f9c2e0.png)
I can't debug the code because I don't have enough GPU. If I have one image, one image has five captions. My task is to predict the similarity between one image and five different subtitles, can I change it to a multi-classification problem. If I predict their similarity score, can I usel1loss be used here?

**Comment by LiJunnan1992** [comment]
You can change the loss according to your specific tasks
